### DISCUSSION TITLE
How Strimzi Kafka Debezium and PG Restore Works

### DISCUSSION THREAD

**Discussion by bahaeddinoz** [strimzi/strimzi-kafka-operator#8647] [Q&A]
Hi All,
We use Strimzi Kafka in production with debezium as producer for postgresql db and our aplication is consumer. Strimzi kafka is the heart of the design. I just confused to understand how postgresql restore works when we need it. I would be grateful if you can help me to understand. 
The Scenario;
- Let's say i need to restore PG db to the point of 1 hour ago how should i handle the all steps? Debezium also understand all restoring changes and going to push to kafka i think. I just want to go that specific time on db and keep the kafka as it is. What is the best way to do it ? 

Thank you for your help in advance!

**Comment by bahaeddinoz** [comment]
any thoughts :( ?

**Reply by scholzj** [reply]
TBH, I do not think Strimzi really interferes with any of this - so this is more a Debezium question than Strimzi question. So maybe asking it in some Debezium forum gives you a better chance for a response. Or maybe I did not understand how Strimzi fits into this.
